Projects/PhoenixPE/Components/.NET Runtime/DotNET9.script

```diff
@@ -30,20 +30,20 @@
 
 [Main]
 Title=.NET 9
-Description=(v9.0.0) .NET 9 is a cross-platform software development framework for building and running applications.
+Description=(v9.0.2) .NET 9 is a cross-platform software development framework for building and running applications.
 Selected=False
 Level=4
 Author=Homes32
-Version=1.0.0.0
-Date=2024-12-14
+Version=1.0.1.0
+Date=2025-02-11
 Mandatory=False
 
 [Variables]
 %ProgramTitle%=".NET 9 Desktop Runtime"
 %ProgramFolder%="dotnet"
-%ProgramVersion%="9.0.0"
-%DownloadURLx86%=https://download.visualstudio.microsoft.com/download/pr/8dfbde7b-c316-418d-934a-d3246253f342/69c6a35b77a4f01b95588e1df2bddf9a/windowsdesktop-runtime-9.0.0-win-x86.exe
-%DownloadURLx64%=https://download.visualstudio.microsoft.com/download/pr/685792b6-4827-4dca-a971-bce5d7905170/1bf61b02151bc56e763dc711e45f0e1e/windowsdesktop-runtime-9.0.0-win-x64.exe
+%ProgramVersion%="9.0.2"
+%DownloadURLx86%=https://builds.dotnet.microsoft.com/dotnet/WindowsDesktop/9.0.2/windowsdesktop-runtime-9.0.2-win-x86.exe
+%DownloadURLx64%=https://builds.dotnet.microsoft.com/dotnet/WindowsDesktop/9.0.2/windowsdesktop-runtime-9.0.2-win-x64.exe
```

Comment:
Since MS seems changed the url scheme maybe we can just put the version var into the links?